book/pipelines.md

```diff
@@ -353,34 +353,10 @@ In this case, sleep takes `nothing` and instead expects an argument.
 So, we can supply the output of the `echo` command as an argument to it:
 `echo 1sec | sleep $in` or `sleep (echo 1sec)`
 
-## Behind the Scenes
+## Result Display Rendering
```

Comment:
```suggestion
## Rendering Display Results
```

Reply:
I think I prefer this headline either way; no matter if the hook is the final renderer or not.

Although only one of the other 14 headlines uses a verb form like this.